srcpkgs/mdcat/template

```diff
@@ -1,15 +1,26 @@
 # Template file for 'mdcat'
 pkgname=mdcat
-reverts="0.22.2_1"
-version=0.22.1
-revision=3
+version=0.23.0
+revision=1
 wrksrc="${pkgname}-${pkgname}-${version}"
 build_style=cargo
-hostmakedepends="cmake pkg-config"
+hostmakedepends="pkg-config ruby-asciidoctor"
 makedepends="openssl-devel"
 short_desc="Cat for markdown"
 maintainer="Wilson Birney <wpb@360scada.com>"
 license="MPL-2.0"
 homepage="https://github.com/lunaryorn/mdcat"
 distfiles="https://github.com/lunaryorn/mdcat/archive/mdcat-${version}.tar.gz"
-checksum=b14846542fb8b60ac0235b399136372df7569aa59ed63f3faf88ff7a485abe5f
+checksum=9fba4fe06f8970a70ca75666f8245181583a6ba5fb11536ae7e572f74c9d1f4c
+
+post_install() {
+	bash_completion=$(find ${wrksrc}/target -name mdcat.bash -print -quit)
+	vcompletion ${bash_completion} bash
+	zsh_completion=$(find ${wrksrc}/target -name _mdcat -print -quit)
+	vcompletion ${zsh_completion} zsh
+	fish_completion=$(find ${wrksrc}/target -name mdcat.fish -print -quit)
+	vcompletion ${fish_completion} fish
+
+	man_page=$(find ${wrksrc}/target -name mdcat.1 -print -quit)
+	vman ${man_page}
```

Comment:
Dirty deeds done dirt cheap :p

Reply:
It's cargo's fault to dump things deep inside /target/release/build xD